website/markdown/docs/usage/adoption-guidelines.mdx

```diff
@@ -0,0 +1,89 @@
+---
+name: Adoption guidelines
+excerpt: This page contains guidelines for migrating existing Xcode projects to Tuist.
+---
+
+# Adoption guidelines
+
+Unless you bootstrap a new project using Tuist,
+in which case you get everything configured automatically,
+you'll have to define your Xcode projects using Tuist's primitives _(e.g. Project.swift and Workspace.swift files)_.
+How tedious this process is,
+depends on how complex your projects are.
+
+As you probably know,
```

Comment:
I'd rephrase this one, with something more mild, like: "as you probably know, Xcode projects can become very complex in time"